### PR TITLE
Solves issue#7: Add new classic.mp3 and trust in me.mp3

### PR DESCRIPTION
**Pull request by brownsands**
issue #7

### CODE REVIEW

**Comment by brownsands** [comment]
Add new classic.mp3 and trust in me.mp3

File: index.html [hunk 1]
```diff
@@ -297,7 +297,7 @@ <h2><i>Kiss And Tell</i></h2>
 
             <a href="#" class="list-group-item">Trust In Me</a>
              <audio controls>
- 			<source src="songs/Kygo Selena Gomez - It Aint Me (with Selena Gomez) (Audio).mp3"
+ 			<source src="songs/Made_in_Paris_Trust_Me_Lyndon_Kidd_Remix_(mp3co.info)"
```
Comment (OrionStar25) [inline index.html]: This song is wrong. You have to add `Trust in Me by Selena Gomez`.

**Review by OrionStar25 — CHANGES_REQUESTED** [review]
You're on the right track, but please add a working live link of your fork. Read the `readme`. :+1:

**Comment by brownsands** [comment]
https://brownsands.github.io/Selena-Gomez-Discography/

**Review by OrionStar25 — APPROVED** [review]
Great work! :fire:

**Pull request merged**